src/modules/commander/accelerometer_calibration.cpp

```diff
@@ -391,53 +366,34 @@ int do_accel_calibration(orb_advert_t *mavlink_log_pub)
 			}
 
 			// Ensure the calibration values used by the driver are at default settings when we are using thermal calibration data
-			accel_scale.x_offset = 0.f;
-			accel_scale.y_offset = 0.f;
-			accel_scale.z_offset = 0.f;
-			accel_scale.x_scale = 1.f;
-			accel_scale.y_scale = 1.f;
-			accel_scale.z_scale = 1.f;
+			x_offset = 0.f;
+			y_offset = 0.f;
+			z_offset = 0.f;
+			x_scale = 1.f;
+			y_scale = 1.f;
+			z_scale = 1.f;
 		}
 
 		// save the driver level calibration data
 		(void)sprintf(str, "CAL_ACC%u_XOFF", uorb_index);
-		failed |= (PX4_OK != param_set_no_notification(param_find(str), &(accel_scale.x_offset)));
+		failed |= (PX4_OK != param_set_no_notification(param_find(str), &(x_offset)));
```

Comment:
I always call out using bitwise or instead of the normal `||` for bools.

Reply:
I'm not following, example?